erdpy/accounts.py

```diff
@@ -1,6 +1,7 @@
 import logging
 from pathlib import Path
 from typing import Any, Optional
+from getpass import getpass
```

Comment:
Nice :+1: (only now I learn that this module is in the standard library - it seems we also use it in `cli_wallet.py`).